app/controllers/api/v2/app_groups_controller.rb

```diff
@@ -59,6 +59,78 @@ def cluster_templates
     render json: cluster_templates
   end
 
+  def profile_app
+    profiles = []
+    AppGroup.active.all.each do |appGroup|
+      environment = appGroup.environment
+      helm_infra = appGroup.helm_infrastructure
+
+      infra_values = helm_infra.values.to_json
+      object = JSON.parse(infra_values, object_class: OpenStruct)
+
+      replication_factor = environment == "production" ? 2 : 1
+
+      barito_apps =[]
+      appGroup.barito_apps.where(status:"ACTIVE").each do |barito_app|
+        days = barito_app.log_retention_days
+        if days == nil
+          days = environment == "production" ? 14 : 7
```

Comment:
if days == nil, you can refer to app_group.log_retention_days value

Reply:
Yes mas, sounds about right, will change this one mas

cc @Abhinav1299

Reply:
Done mas